### PR TITLE
Us/3: Un utilisateur connecté peut créer un nouvel événement temporaire

### PR DESCRIPTION
**Pull request by AamuLumi**

### CODE REVIEW

File: src/components/LeftBar/LeftBar.scss [hunk 1]
```diff
@@ -0,0 +1,178 @@
+.left-bar-container {
+	.left-bar {
+		width: 399px;
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: 400px

File: src/components/LeftBar/LeftBar.scss [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+		height: 1945px;
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: Remplacer par `height: 100%` ou similaire

File: src/components/LeftBar/LeftBar.scss [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+	.items {
+		padding-top: 6%;
+		padding-left: 5.8%;
+		padding-right: 6%;
+	}
+
+	.new-events {
+		width: 352px;
+		height: 43px;
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: Remplacer width et height fixes par 
```
width: 100%;
padding: XXpx;
```

File: src/components/LeftBar/LeftBar.scss [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 0.27px;
+		text-align: center;
+		color: #f9f9f9;
+		align-items: center;
+		margin-top: 25px;
+		text-decoration: none;
+		box-sizing: border-box;
+		padding-left: 24.6%;
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: `padding-left: 24.6%` ?

File: src/components/LeftBar/LeftBar.scss [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+	}
+
+	.new-talk {
+		width: 352px;
+		height: 43px;
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: Même commentaire que pour les events

File: src/components/LeftBar/LeftBar.scss [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+		z-index: 3;
+	}
+
+	.items {
+		padding-top: 6%;
+		padding-left: 5.8%;
+		padding-right: 6%;
+	}
+
+	.new-events {
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: => `new-event`

File: src/components/LeftBar/LeftBar.scss [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+		align-items: center;
+		display: flex;
+		z-index: 3;
+
+		&:hover {
+			cursor: pointer;
+		}
+	}
+
+	.new-events-text {
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: => `new-event-text`

File: src/components/LeftBar/LeftBar.scss [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 0.27px;
+		text-align: center;
+		color: #f9f9f9;
+		align-items: center;
+		margin-top: 25px;
+		text-decoration: none;
+		box-sizing: border-box;
+		padding-left: 32.1%;
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: `padding-left: 32.1%;` ?

File: src/components/LeftBar/LeftBar.scss [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+		font-size: 18px;
+		font-weight: normal;
+		font-style: normal;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 1.03px;
+		color: #f0f0f0;
+	}
+
+	.my-event2 {
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: `.my-event2`à supprimer

File: src/components/LeftBar/LeftBar.scss [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+		font-size: 18px;
+		font-weight: normal;
+		font-style: normal;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 1.03px;
+		color: #f0f0f0;
+	}
+
+	.my-talks2 {
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: `.my-talks2` à supprimer

File: src/components/LeftBar/LeftBar.scss [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+		font-size: 18px;
+		font-weight: normal;
+		font-style: normal;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 1.03px;
+		color: #f0f0f0;
+	}
+
+	.justifie {
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: `.justifie` ?

File: src/components/LeftBar/LeftBar.scss [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+		font-size: 21px;
+		font-weight: 300;
+		font-style: italic;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 3.37px;
+		color: #f0f0f0;
+	}
+
+	.my-talks {
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: => `.talk`

File: src/components/LeftBar/LeftBar.scss [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+		font-size: 21px;
+		font-weight: 300;
+		font-style: italic;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 3.37px;
+		color: #f0f0f0;
+	}
+
+	.my-event {
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: => `.event`

File: src/components/Header/index.tsx [hunk 14]
```diff
@@ -1,20 +1,66 @@
...
+						className={`github ${this.state.id === 'github' ? 'selected' : ''}`}
+						data-page="github"
+						onClick={this.elementsClick}>
+						GitHub
+					</Link>
+
+					<div className="auth">
+						<div className="authname">
+							<div className="username">Harry Potter</div>
+							<div className="pseudoname">@harry</div>
```
Comment (AamuLumi) [inline src/components/Header/index.tsx]: => `.username`

File: src/components/Header/index.tsx [hunk 15]
```diff
@@ -1,20 +1,66 @@
...
+						to="/github/"
+						className={`github ${this.state.id === 'github' ? 'selected' : ''}`}
+						data-page="github"
+						onClick={this.elementsClick}>
+						GitHub
+					</Link>
+
+					<div className="auth">
+						<div className="authname">
+							<div className="username">Harry Potter</div>
```
Comment (AamuLumi) [inline src/components/Header/index.tsx]: => `.full-name`

File: src/components/Header/index.tsx [hunk 16]
```diff
@@ -1,20 +1,66 @@
...
+							<div className="pseudoname">@harry</div>
+						</div>
+						<img src={HP} className="image" />
```
Comment (AamuLumi) [inline src/components/Header/index.tsx]: => `.avatar`

File: src/components/LeftBar/LeftBar.scss [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+		font-size: 21px;
+		font-weight: bold;
+		font-style: normal;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 0.27px;
+		text-align: center;
+		color: #f9f9f9;
+		align-items: center;
+		margin-top: 25px;
```
Comment (AamuLumi) [inline src/components/LeftBar/LeftBar.scss]: Replace with padding

File: src/components/SearchBody/SearchBody.scss [hunk 18]
```diff
@@ -0,0 +1,136 @@
...
+	.contain {
+		display: flex;
+		justify-content: center;
+		flex-direction: column;
+		align-items: center;
+		z-index: 3;
+		width: 100%;
+	}
+
+	.container {
```
Comment (AamuLumi) [inline src/components/SearchBody/SearchBody.scss]: Difference between `.contain` and `.container` ?

File: src/pages/FormOne/FormOne.scss [hunk 19]
```diff
@@ -0,0 +1,472 @@
...
+	.end,
+	.places {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		flex-wrap: nowrap;
+		width: 33%;
+	}
+
+	.input-startDate,
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: No camelCase in css

File: src/pages/FormOne/FormOne.scss [hunk 20]
```diff
@@ -0,0 +1,472 @@
...
+	.input-endDate,
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: No camelCase in css

File: src/pages/FormOne/FormOne.scss [hunk 21]
```diff
@@ -0,0 +1,472 @@
...
+		font-weight: normal;
+		font-style: normal;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 0.3px;
+		color: #ffffff;
+		margin-right: 2%;
+	}
+
+	.fullAddress-inputs {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: No camelCase in css

File: src/pages/FormOne/FormOne.scss [hunk 22]
```diff
@@ -0,0 +1,472 @@
...
+		margin-right: 32px;
+		width: 100%;
+	}
+
+	.adress {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: => `.address`

File: src/pages/FormOne/FormOne.scss [hunk 23]
```diff
@@ -0,0 +1,472 @@
...
+			font-weight: normal;
+			font-style: normal;
+			font-stretch: normal;
+			line-height: normal;
+			letter-spacing: 0.3px;
+			color: #ffffff;
+		}
+	}
+
+	.input-short-descriptions {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: => `.input-short-description`

File: src/pages/FormOne/FormOne.scss [hunk 24]
```diff
@@ -0,0 +1,472 @@
...
+		}
+	}
+
+	.textarea-long-description {
+		label {
+			margin-top: 2.7%;
+		}
+		textarea {
+			width: 100%;
+			height: 128px;
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Don't force textarea height. Instead, use `numberOfLines` html property to define base size

File: src/pages/FormOne/FormOne.scss [hunk 25]
```diff
@@ -0,0 +1,472 @@
...
+			border-top: 0px;
+			border-left: 0px;
+			border-right: 0px;
+			background: transparent;
+			margin-top: 0.7%;
+		}
+	}
+
+	.input-ticketing-url {
+		input {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Code duplication

File: src/pages/FormOne/FormOne.scss [hunk 26]
```diff
@@ -0,0 +1,472 @@
...
+			border-top: 0px;
+			border-left: 0px;
+			border-right: 0px;
+			background: transparent;
+			margin-top: 0.7%;
+		}
+	}
+
+	.input-event-url {
+		input {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Code duplication

File: src/pages/FormOne/FormOne.scss [hunk 27]
```diff
@@ -0,0 +1,472 @@
...
+			border-top: 0px;
+			border-left: 0px;
+			border-right: 0px;
+			background: transparent;
+			margin-top: 0.5%;
+		}
+	}
+
+	.input-color-secondary {
+		input {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Code duplication

File: src/pages/FormOne/FormOne.scss [hunk 28]
```diff
@@ -0,0 +1,472 @@
...
+		margin-right: 3.6%;
+		width: 100%;
+	}
+
+	.color-secondary {
+		width: 100%;
+	}
+
+	.input-color-primary {
+		input {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Code duplication

File: src/pages/FormOne/FormOne.scss [hunk 29]
```diff
@@ -0,0 +1,472 @@
...
+			background: transparent;
+			margin-top: 0.5%;
+		}
+	}
+
+	.color-gradient {
+		width: 100%;
+	}
+
+	.color-gradient-to-event {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Code duplication

File: src/pages/FormOne/FormOne.scss [hunk 30]
```diff
@@ -0,0 +1,472 @@
...
+	.ticketing-url {
+		margin-top: 2.1%;
+	}
+
+	.preview-logo-block {
+		margin-left: 2.2%;
+		width: 50%;
+	}
+
+	.preview-logo-text {
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Code duplication

File: src/pages/FormOne/FormOne.scss [hunk 31]
```diff
@@ -0,0 +1,472 @@
...
+		margin-bottom: 5.6%;
+		width: 100%;
+		flex-wrap: wrap;
+		justify-content: flex-end;
+		margin-top: 4.2%;
+	}
+
+	.save-event {
+		margin-top: 6%;
+		width: 352px;
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: No fixed width => Use padding

File: src/pages/FormOne/FormOne.scss [hunk 32]
```diff
@@ -0,0 +1,472 @@
...
+		height: 43px;
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: No fixed height => Use height

File: src/pages/FormOne/FormOne.scss [hunk 33]
```diff
@@ -0,0 +1,472 @@
...
+
+		&:hover {
+			cursor: pointer;
+		}
+	}
+
+	.submit-event {
+		margin-top: 6%;
+		width: 352px;
+		height: 43px;
```
Comment (AamuLumi) [inline src/pages/FormOne/FormOne.scss]: Same comments than previous button

File: src/pages/FormOne/index.tsx [hunk 34]
```diff
@@ -0,0 +1,328 @@
...
+
+	private onColorChanged = (e: Event<string>) => {
+		this.onValueChanged({
+			name: e.name,
+			value: e.value.charAt(0) === '#' ? e.value : `#${e.value}`,
+		});
+	};
+
+	private onTopicsChanged = (e) => {
+		if (e.value === 13) {
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: Magic number: extract 13 in a named constant

File: src/pages/FormOne/index.tsx [hunk 35]
```diff
@@ -0,0 +1,328 @@
...
+			this.onMultipleValuesChanged({
+				topics: newTopic,
+				currentTopic: '',
+			});
+		}
+	};
+
+	private onTopicsDelete = (e) => {
+		const array = [...this.state.formData.topics];
+		const index = parseInt(e.currentTarget.dataset.id, 16);
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: Base 16? What?

File: src/pages/FormOne/index.tsx [hunk 36]
```diff
@@ -0,0 +1,328 @@
...
+			},
+		};
+	}
+
+	handleSubmit(event: { preventDefault: () => void }) {
+		event.preventDefault();
+	}
+
+	public render() {
+		console.log(this.state.formData);
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: Remove console.log

File: src/pages/FormOne/index.tsx [hunk 37]
```diff
@@ -0,0 +1,328 @@
...
+			value: array,
+		});
+	};
+
+	public getInitialState() {
+		return {
+			formData: {
+				beginColor: '#',
+				endColor: '#',
+				longitude: ' ',
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: Must be a number

File: src/pages/FormOne/index.tsx [hunk 38]
```diff
@@ -0,0 +1,328 @@
...
+				latitude: ' ',
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: Must be a number

File: src/pages/FormOne/index.tsx [hunk 39]
```diff
@@ -0,0 +1,328 @@
...
+										<div className="zone-of-preview">
+											<img
+												src={this.state.formData.urllogoabsolute}
+												className="preview-logo"
+											/>
+										</div>
+									</div>
+								</div>
+
+								<div className="button-event">
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: => `.buttons-container`

File: src/pages/FormOne/index.tsx [hunk 40]
```diff
@@ -0,0 +1,328 @@
...
+									<div className="left-button">
+										<div className="save-event">Sauvegarder l’événement</div>
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: Use a HTML button for a11y

File: src/pages/FormOne/index.tsx [hunk 41]
```diff
@@ -0,0 +1,328 @@
...
+									</div>
+									<div className="right-button">
+										<div className="submit-event">Soumettre l’événement</div>
```
Comment (AamuLumi) [inline src/pages/FormOne/index.tsx]: Use a HTML button for a11y

File: src/pages/FormOne/index.tsx (hunk 37 above)
Comment (capelliludivine) [reply src/pages/FormOne/index.tsx]: => avoid initial value in input

File: src/pages/FormOne/index.tsx (hunk 38 above)
Comment (capelliludivine) [reply src/pages/FormOne/index.tsx]: => avoid initial value in input

File: src/components/LeftBar/LeftBar.scss (hunk 11 above)
Comment (capelliludivine) [reply src/components/LeftBar/LeftBar.scss]: justifie les éléments (comme dans google docs)

**Pull request merged**